autotest/test_util_2d_and_3d.py

```diff
@@ -614,3 +614,40 @@ def test_mflist_fromfile(function_tmpdir):
     flx_array = wel.stress_period_data.array["flux"][0]
     for k, i, j, flx in zip(wel_data.k, wel_data.i, wel_data.j, wel_data.flux):
         assert flx_array[k, i, j] == flx
+
+
+def test_pathlib_support():
+    import tempfile
+    from pathlib import Path
+
+    with tempfile.TemporaryDirectory() as tmpdir:
+        model_ws = Path(tmpdir)
+        out_well_spd = {0: model_ws.joinpath("wel.txt")}
+        with open(out_well_spd[0], "w") as f:
+            f.write("0 2 2 -10\n")
+            f.write("0 1 2 -10\n")
+        m = Modflow("test_model", model_ws=model_ws, exe_name="mf2005")
+        wel = ModflowWel(
+            m,
+            stress_period_data=out_well_spd,
+            ipakcb=740,
+        )
+        m.write_input()
+
+
+def test_open_close_one_val():
```

Comment:
Similar to previous:
```suggestion
def test_open_close_one_val(tmp_path):
```
then rewrite to use fixture.